src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -144,8 +145,8 @@ public async Task<OrchestratorResponse<EmployerTeamMembersViewModel>> ChangeRole
         public async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAccount(string accountId, string externalUserId)
         {
             try
-            {
-                var apiGetAccountTask = _accountApiClient.GetAccount(accountId);
+            {
+                var apiGetAccountTask = await _accountApiClient.GetAccount(accountId).ConfigureAwait(false);
```

Comment:
do we need to do this before firing off the other tasks? This should be an unlikely scenario so would the benefits of parallelising most of the time outweigh the edge case where we are doing unnecessary work ?